packages/amplication-data-service-generator/src/generate-code.ts

```diff
@@ -63,6 +65,8 @@ async function writeModules(
   console.info(`Writing modules to ${destination} ...`);
   await Promise.all(
     modules.map(async (module) => {
+      if (filesToFilter.test(module.path)) return;
```

Comment:
I dont think we should add any logic at this layer. If the DSG decided to includes any files in the results, they should be included. 
I think we can add this filter as part of the util "CopyStaticFiles" - and should document that behavior